projects/sunbird-player-sdk/src/lib/player-utils/service/content-compability/content-compability.service.ts

```diff
@@ -13,9 +13,9 @@ export class ContentCompabilityService {
       const  compatibilityError = new Error();
       compatibilityError.message = `Player supports ${this.playerContentCompatibiltyLevel} but content compatibility is ${currentCompatibilityLevel}`;
       compatibilityError.name = 'contentCompatibily';
-      return compatibilityError;
+      return { error: compatibilityError , isCompatibilityError: true };
```

Comment:
change **isCompatibilityError** to **isCompatible** and update the test cases as well